addons/host-upgrades/addon.rb

```diff
@@ -5,16 +5,48 @@
   license 'Apache License 2.0'
 
   config {
-    attribute :interval, Pharos::Types::String
```

Comment:
If we wanted to make the 1.2 upgrade smoother, we could continue to accept existing addon configurations with `interval`, and just translate that to `schedule: "@every ..."`.

Reply:
Not sure if it's needed if the new addon complains when schedule is missing?

Reply:
Everyone using the existing `host-upgrades` addon will need to edit their `cluster.yml` when upgrading to 1.2. That could be avoided.

Reply:
I think it's ok to edit cluster.yml in this case.